default.nix

```
@@ -80,7 +80,7 @@ let
       dir = pkgs.runCommand name
       {
         inherit imageDigest;
-        impureEnvVars = pkgs.lib.fetchers.proxyImpureEnvVars;
+        impureEnvVars = pkgs.lib.fetchers.proxyImpureEnvVars ++ ["SRC_REGISTRY_USER" "SRC_REGISTRY_PASSWORD"];
```

What about adding the `NIX2CONTAINER_` prefix in order to explicit show where this environment variables are used.
```suggestion
        impureEnvVars = pkgs.lib.fetchers.proxyImpureEnvVars ++ ["NIX2CONTAINER_SRC_REGISTRY_USER" "NIX2CONTAINER_SRC_REGISTRY_PASSWORD"];
```